pegasus/sites.v3/hourofcode.com/views/course_carousel.haml

```diff
@@ -29,7 +29,7 @@
                   = tutorial[:platformtext]
                 %p.slide-longdescription= tutorial[:longdescription]
               .content-footer
-                %a.link-button{href: url, target: "_blank"}
+                %a.link-button{href: url, target: "_blank", aria: {label: tutorial[:buttonarialabel]}}
```

Comment:
It's not as great of a solution as the one you've got, but would it be possible to do something like `aria: {label: "Go to #{tutorial[:name]"}` here instead to save the database migration?

Reply:
Oh that's a good idea…

Reply:
I considered that at first but was concerned about some of the overlapping names (particularly with Code.org's names). If this is okay to have a bit of overlap I can for sure do that and would be able to test it!

Reply:
Actually looking more into it I think I may have jumped the gun with the concern of overalpping names because the overlapping ones would appear in different sections (of elementary, middle, and high schools) so I'll switch to that plan!